src/processPattern.js

```diff
@@ -1,11 +1,10 @@
-import Promise from 'bluebird';
+import globby from 'globby';
+import createThrottle from 'async-throttle';
```

Comment:
[p-all](https://github.com/sindresorhus/p-all)
[p-limit](https://github.com/sindresorhus/p-limit)
[p-throttle](https://github.com/sindresorhus/p-throttle)

[async-throttle](https://github.com/zeit/async-throttle) is **deprecated**

Reply:
@michael-ciniawsky archived, not deprecated :+1:

Reply:
None the less I agree with @michael-ciniawsky , i'd prefer using something that is actively maintained